src/roles/check_database_connection/tasks/main.yaml

```diff
@@ -0,0 +1,17 @@
+---
+- name: Check Foreman database connectivity and credentials
+  community.postgresql.postgresql_ping:
+    login_host: "{{ foreman_database_host }}"
+    login_user: "{{ foreman_database_user }}"
+    login_password: "{{ foreman_database_password }}"
+    login_db: "{{ foreman_database_name }}"
```

Comment:
This currently gets executed for local DB too, but the user and the db don't exist yet -- they are created later by the `postgresql` role. So we probably should skip this check for local DB alltogether?